### PR TITLE
data:ElvisJin

### PR DESCRIPTION
**Pull request by ElvisJin**
Hello, I just added my profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5009"></a>

### CODE REVIEW

**Review by mahfuztbt — APPROVED** [review]
LGTM

**Review by samyabrata-maji — DISMISSED** [review]

File: data/ElvisJin.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Elvis Jin",
+  "type": "personal",
+  "bio": "Over 7 years of experience delivering tech solutions. Avid programmer and fast learner. Obsessed with detail and quality. Always trying to improve development and communication processes. I love simple and elegant solutions for every problem.",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/elvis-j-934261266",
+      "icon": "FaLinkedIn"
```
Comment (swagataroy30) [inline data/ElvisJin.json:9]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by swagataroy30 — COMMENTED** [review]
Fix the typo, the icon name should be `FaLinkedin`. Thank You!

**Review by Hrishi5111998 — DISMISSED** [review]

**Review by SurajPratap10 — APPROVED** [review]

File: data/ElvisJin.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "name": "GitHub",
+      "url": "https://www.github.com/ElvisJin",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "CodeSignal",
+      "url": "https://app.codesignal.com/profile/elvis_jin",
+      "icon": "FaBrowser"
```
[the remark this answers is not in the document]
Comment (kumarsonsoff3) [reply data/ElvisJin.json:19]: In this case, the default fallback icon `FaGlobe` will be used. Later they can update it in another PR if they want!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**